tests/e2e/setup.go

```diff
@@ -28,16 +26,14 @@ import (
 // Any method implemented for this struct will be ran when suite.Run() is called.
 type CCVTestSuite struct {
 	suite.Suite
-	coordinator       *ibctesting.Coordinator
-	providerChain     *ibctesting.TestChain
-	consumerChain     *ibctesting.TestChain
-	providerApp       e2e.ProviderApp
-	consumerApp       e2e.ConsumerApp
-	providerClient    *ibctmtypes.ClientState
```

Comment:
removed these two confusingly named parameters in favor of more explicit inline assertions